tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -984,7 +984,7 @@ Status IrEmitterUnnested::EmitScatter(
     llvm::Value* input_address = llvm_ir::EmitAllocaAtFunctionEntry(
         llvm_ir::PrimitiveTypeToIrType(updates->shape().element_type(),
                                        module_),
-        "input_address", &b_);
+        absl::string_view(), &b_);
```

Comment:
`"input_address"` shall be better kept for better debugging.
Multiple `alloca` will be automatically sequenced.

Reply:
I can put the `"input_address"` if need be. 
I removed it because this name was `nullpttr` before PR #1003, and since I am not introducing new names in all the other instances, wanted to keep this once consistent with the rest (for upstreaming purposes)

Reply:
As multiple `Alloca` would be moved to the entry basic block I think it's a better idea to name them so at least some traces could be pointed back to `EmitScatter` here.